lib/DB_functions/get_id_and_insert_active_data.sql

```diff
@@ -0,0 +1,10 @@
+-- Return the next available id for a given zobjtype and insert a new active data record
+CREATE OR REPLACE FUNCTION get_id_and_insert_active_data(obj_name varchar) RETURNS text AS $$
```

Comment:
Nice utility function!

Reply:
Haha, yeah sometimes it's the little things. There are probably some sql scripts in our codebase that could be made more readable with this. I was thinking if we need to get_id and reference it 3 or more times in a script, we can use session variables (https://blog.mclaughlinsoftware.com/2019/09/28/session-variables/)